### ISSUE TITLE
Password loop

### ISSUE THREAD

**Issue by nedludd** [sipcapture/homer-installer#8]
Installing on Centos 7.1 (tried on 7.2 as well) and while the "homer-installer-5.0.5-1.x86_64" is being installed I get prompted for "New password" over and over.

Was prompted for "New password" and "Re-enter new password" over 100 times before I gave up.

**Comment by lmangani** [comment]
The CentOS setup should provide the root password to use as part of the prompt, while any other custom password must comply in complexity with the security settings of MySQL which are quite something in the recent versions - the output is not always helpful, so If you have a setup log we can confirm so.

**Comment by nedludd** [comment]
Hmm, I got the installer to complete at least.  What I did was take the password from the log, in this case:

`Starting mysql secure installation [ 2zsh??pFC.<8 ]` 

And enter it at the prompt. This let the installer complete.  There are other errors, as you'll see in the attached log.   The UI starts up, but I'm not sure if anything will work.
[homer_installer.log.txt](https://github.com/sipcapture/homer-installer/files/450614/homer_installer.log.txt)
